frontend/sentiment-dashboard/src/app/shared/nav/nav.component.html

```diff
@@ -23,3 +29,15 @@
       </ul>
     </div>
   </nav>
+
+<ng-template #template>
+  <div class="modal-header">
+    <h4 class="modal-title pull-left">Change password</h4>
```

Comment:
Use `h2` to keep headers in order for better screen reader compatibility - you can add `class="h4"` if you want to keep the text-size of a level 4 header.